static/src/App.vue

```diff
@@ -66,12 +76,18 @@ export default defineComponent({
         id="login-view"
         @login="login"
       ></LoginView>
+      <TasksOverView
+        v-else-if="page == 'tasks'"
+        id="tasks-overview"
+        @open-task="openTask"
+      ></TasksOverView>
       <TaskView
-        v-else
+        v-else-if="page == 'task' && task"
```

Comment:
'task' && task", this confuses me somehow. maybe this is a leftover or does it require the check for a string and also an object?

Reply:
Good question, I'll take that to #127 for discussion